### PR TITLE
[jit] Fix __constants__ for some nn modules

### LINKED ISSUE #20978: Cannot print LayerNorm in ScriptModule
## 🐛 Bug
Version: master (1.2.0.dev20190526)
Got KeyError: 'elementwise_affine' when printing a ScriptModule with a LayerNorm in it.

## To Reproduce
```python
import torch
from torch.nn import LayerNorm
from torch.jit import ScriptModule


class Test(ScriptModule):
    def __init__(self, dim):
        super().__init__()
        self.layer_norm = LayerNorm(dim)


if __name__ == '__main__':
    m = Test(100)
    print(m)
```

The result is:
```
raceback (most recent call last):
  File "/home/chenyufei/Development/nn-parser/local_scripts/pytorch_1_2_master_bug.py", line 14, in <module>
    print(m)
  File "/home/chenyufei/.local/anaconda3.7/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1044, in __repr__
    mod_str = repr(module)
  File "/home/chenyufei/.local/anaconda3.7/lib/python3.7/site-packages/torch/nn/modules/module.py", line 1038, in __repr__
    extra_repr = self.extra_repr()
  File "/home/chenyufei/.local/anaconda3.7/lib/python3.7/site-packages/torch/nn/modules/normalization.py", line 161, in extra_repr
    'elementwise_affine={elementwise_affine}'.format(**self.__dict__)
KeyError: 'elementwise_affine'
```

## Expected behavior
In stable version, it is:
```
Test(
  (layer_norm): WeakScriptModuleProxy()
)
```

## Environment
Collecting environment information...
PyTorch version: 1.2.0.dev20190526
Is debug build: No
CUDA used to build PyTorch: 9.0.176

OS: Ubuntu 18.04.1 LTS
GCC version: (Ubuntu 6.5.0-2ubuntu1~18.04) 6.5.0 20181026
CMake version: version 3.10.2

Python version: 3.7
Is CUDA available: Yes
CUDA runtime version: Could not collect
GPU models and configuration: 
GPU 0: GeForce GTX 980 Ti
GPU 1: GeForce GTX TITAN X

Nvidia driver version: 390.116
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.1.4

Versions of relevant libraries:
[pip3] msgpack-numpy==0.4.3.1
[pip3] numpy==1.16.2
[pip3] numpydoc==0.8.0
[pip3] pytorch-pretrained-bert==0.6.1
[pip3] torch==1.0.0
[pip3] torchfile==0.1.0
[pip3] torchtext==0.4.0
[pip3] torchvision==0.2.1
[conda] Could not collect

### PR DESCRIPTION
**Pull request by driazati**
A bunch of modules were missing entries for `__constants__` which was making their `__repr__`s not work. Others had `__constants__` that were not necessary since it was provided by some parent class instead.

Fixes #20978

Differential Revision: [D15539518](https://our.internmc.facebook.com/intern/diff/15539518/)

### CODE REVIEW

File: torch/nn/modules/instancenorm.py [hunk 1]
```diff
@@ -4,9 +4,6 @@
 
 
 class _InstanceNorm(_BatchNorm):
-    __constants__ = ['running_mean', 'running_var', 'weight', 'bias',
```
Comment (eellison) [inline torch/nn/modules/instancenorm.py]: why were these all removed ?

File: torch/nn/modules/conv.py [hunk 2]
```diff
@@ -478,10 +480,6 @@ def forward(self, input):
 
 @weak_module
 class _ConvTransposeMixin(object):
-    __constants__ = ['stride', 'padding', 'kernel_size', 'dim_size',
```
Comment (eellison) [inline torch/nn/modules/conv.py]: same ?

**Review by eellison — APPROVED** [review]
Good if tests pass

File: torch/nn/modules/instancenorm.py (hunk 1 above)
Comment (driazati) [reply torch/nn/modules/instancenorm.py]: They're on the parent class

**Pull request closed without merging**